### ISSUE TITLE
Missing key words 'lang_cap' and 'logits' in GRPO traning

### ISSUE THREAD

**Issue by jeff2048** [AIGeeksGroup/3D-R1#22]
https://github.com/AIGeeksGroup/3D-R1/blob/ed95e2487f4bc1a00e1b81896b2b982463166bd4/models/rl/grpo_trainer.py#L68C1-L69C49

Can you please check this part?

**Comment by Believeht029** [comment]
Thank you for your feedback, we have updated the code to fix this error.
